2020q2/linuxulator.md

```diff
@@ -0,0 +1,38 @@
+## Linux compatibility layer update ##
+
+Contact: Edward Tomasz Napierala, <trasz@FreeBSD.org>
+
+Work has shifted from cleanups to fixing actual applications.
+Current status is being tracked at
+[Linux app status Wiki page](https://wiki.freebsd.org/LinuxApps).
+Current focus is on applications that don't involve X11, mostly
+because they tend to be easier to test and debug, and the bug fixes
+are not application-specific.
+
+Example problems fixed include buggy madvise(2) handling, uname(2)
+returning wrong results for 32 bit apps, recvmsg(2) and accept(2) being
+broken in some circumstances, missing support for `SO_REUSEPORT`,
+`SO_SNDBUFFORCE`, `SO_RCVBUFFORCE`, and `SO_PROTOCOL`, default open files
+limit being way higher than Linux apps tend to assume, 
+
+There's ongoing work on debugging infrastructure and cleanups, such as
+logging warnings for unrecognized system call paramteres, or adding the
```

Comment:
“parameters” is misspelled.

It might be worth somehow clarifying that this is the debugging framework for linux compatibility, not linux compatibility’s support for debugging Linux processes.